<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QWidget" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>100</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>100</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>20</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>sum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
